code/modules/holiday/holidays.dm

```diff
@@ -640,3 +640,13 @@
 
 /datum/holiday/hebrew/passover/getStationPrefix()
 	return pick("Matzah", "Moses", "Red Sea")
+
+/datum/holiday/pride_week
+	name = "Pride Week"
+	begin_day = 1
+	begin_month = JUNE
+	end_day = 7
+	end_month = JUNE
+
+/datum/holiday/pride_week/getStationPrefix()
+	return pick("Trans Rights Are Human Rights","Stonewall","LGBTQIAP+","Justice For George Floyd","Black Lives Matter","Trans Rights","LGBTQIAP+ Rights", "Pride")
```

Comment:
these are not even proper prefixes and as much as george floyd and BLM are real issues, I don't think they relate to pride week?

Reply:
trans rights are human rights delta environment xiiiiii

Reply:
HOLD THE FUCKING LINE
ISNT THE P IN LGBTQIAP+ FOR PEDOS?

Reply:
no it isn't 
it stands for pansexual
maybe try googling something for once

Reply:
![chrome_ImWJwQv4hc](https://user-images.githubusercontent.com/4081722/83499494-50f9da00-a472-11ea-85f6-c93e8861b674.png)
literally 2 seconds of googling